core/src/main/java/org/apache/carbondata/core/datamap/TableDataMap.java

```diff
@@ -63,6 +75,8 @@
 
   private SegmentPropertiesFetcher segmentPropertiesFetcher;
 
+  private static final Log LOG = LogFactory.getLog(TableDataMap.class);
```

Comment:
We do not use apache-common-logs in carbondata project! Please take care of this

Reply:
ok